mopidy_alsamixer/__init__.py

```diff
@@ -22,6 +22,9 @@ def get_config_schema(self):
         schema = super(Extension, self).get_config_schema()
         schema['card'] = config.Integer(minimum=0)
         schema['control'] = config.String()
+        schema['volmin'] = config.Integer(minimum=0, maximum=100)
```

Comment:
max_volume and min_volume perhaps? Main point being not to abbreviate volume and grouping it together</bikeshed>